operations/host_spawn.go

```diff
@@ -641,3 +641,44 @@ func hostTerminate() cli.Command {
 		},
 	}
 }
+
+func hostRunCommand() cli.Command {
+	const commandFlagName = "command"
+
+	return cli.Command{
+		Name:  "run-command",
```

Comment:
Can this be run-script? A command would imply that it's supposed to be run as a single executable rather than as a script.

Reply:
done